api/test/index.js

```diff
@@ -117,3 +117,120 @@ describe('PUT /people/:id', () => {
 
   // TODO -- add validation for person and cause the api call to fail
 })
+
+let oneEventLink
```

Comment:
not sure there's a way to modularize this - I'm assuming there is but I'm not sure.